package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -766,14 +767,18 @@ def _getOneDeltaPoint(self,universe, repInd, i ,t0, dt):
             valHH += self.lg2(np.dot(unitHHVector0,unitHHVectorp))
             valdip +=  self.lg2(np.dot(unitdipVector0,unitdipVectorp))
             n += 1
-        valOH = valOH/n
-        valHH = valHH/n
-        valdip = valdip/n
-        return (valOH,valHH,valdip)
+
+        if n == 0:
+            return (0,0,0)
```

Comment:
Use [PEP 8](https://www.python.org/dev/peps/pep-0008/) formatting (space after comma) – applies to the whole file.

Note: If you start formatting the whole file according to PEP 8 *do this in a separate commit* so that it is clear that this commit only contains formatting. However, any new code that you're adding/changing should already be formatted to PEP 8.